src/client/css/partials/cancelSubscriptionSurvey.css

```diff
@@ -0,0 +1,42 @@
+[data-partial='cancelSubscriptionSurvey'] {
+  text-align: center;
+}
+
+[data-partial='cancelSubscriptionSurvey'] header, 
+[data-partial='cancelSubscriptionSurvey'] form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+}
+
+[data-partial='cancelSubscriptionSurvey'] form {
+  padding: 0 var(--padding-sm);
+  margin: 0 auto;
+  gap: var(--padding-md);
+  max-width: 400px;
+}
+
+[data-partial='cancelSubscriptionSurvey'] textarea {
+  width: 100%;
+  font-family: var(--body-font);
+  padding: var(--padding-sm);
+  height: 100px;
```

Comment:
personal curiosity: Why the use of explicit DOM `height` vs setting [`rows`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows) attribute on the element?